### PR TITLE
Replace any lib leftout

### PR DESCRIPTION
**Pull request by Alexanderlacuna**
#### Description
This PR deletes any lib that may have been left out or any  lib not currently being used

### CODE REVIEW

File: wqflask/wqflask/templates/base.html [hunk 1]
```diff
@@ -252,7 +252,6 @@
             })
     </script>
     <script src="{{ url_for('js', filename='jquery-cookie/jquery.cookie.js') }}" type="text/javascript"></script>
-    <script type="text/javascript" src="/static/new/js_external/json2.js"></script>
```
Comment (BonfaceKilz) [inline wqflask/wqflask/templates/base.html:255]: **issue:** @Alexanderlacuna how come we have no replacement for this?

File: wqflask/wqflask/templates/correlation_page.html [hunk 2]
```diff
@@ -143,7 +142,8 @@ <h2>Trait: {{ this_trait.name }}
     <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTablesExtensions/buttons/js/buttons.html5.min.js') }}"></script>
     <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTablesExtensions/plugins/sorting/natural.js') }}"></script>
     <script language="javascript" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/js/all.min.js"></script>
-    <script language="javascript" type="text/javascript" src="https://cdn.datatables.net/scroller/2.0.2/js/dataTables.scroller.min.js"></script>
+    <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTablesExtensions/scroller/js/dataTables.scroller.min.js') }}"></script>
```
Comment (BonfaceKilz) [inline wqflask/wqflask/templates/correlation_page.html:145]: @zsloan  We use v2.0.3 in Guix; I however don't see this being a problem.

File: wqflask/wqflask/templates/pair_scan_results.html [hunk 3]
```diff
@@ -67,8 +65,6 @@ <h2>
      <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTables/js/jquery.js') }}"></script>
     <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTables/js/jquery.dataTables.min.js') }}"></script>
     <script language="javascript" type="text/javascript" src="{{ url_for('js', filename='DataTablesExtensions/plugins/sorting/scientific.js') }}"></script>
-    <script language="javascript" type="text/javascript" src="/static/packages/DT_bootstrap/DT_bootstrap.js"></script>
-    <script language="javascript" type="text/javascript" src="/static/packages/TableTools/media/js/TableTools.min.js"></script>
```
Comment (BonfaceKilz) [inline wqflask/wqflask/templates/pair_scan_results.html:71]: +1 for removing these non-existent files.

File: wqflask/wqflask/templates/base.html (hunk 1 above)
Comment (BonfaceKilz) [reply wqflask/wqflask/templates/base.html:255]: I see that this files exists here:  wqflask/wqflask/static/new/js_external/json2.js

**Review by BonfaceKilz — CHANGES_REQUESTED** [review]
@Alexanderlacuna one minor question and this should be good to go.

**Comment by BonfaceKilz** [comment]
I left a minor comment that you should check out.

-- 
Bonface M. K. D4F09EB110177E03C28E2FE1F5BBAE1E0392253F
Humble GNU Emacs User / Bearer of scheme-y parens
Curator: <https://upbookclub.com> / Twitter: @BonfaceKilz

File: wqflask/wqflask/templates/base.html (hunk 1 above)
Comment (Alexanderlacuna) [reply wqflask/wqflask/templates/base.html:255]: @BonfaceKilz  Json is now included in the js language so I don't think we need the library.

**Comment by BonfaceKilz** [comment]
Alexander Kabui <notifications@github.com> writes:

> @Alexanderlacuna commented on this pull request.
>
>
>
>> @@ -252,7 +252,6 @@
>              })
>      </script>
>      <script src="{{ url_for('js', filename='jquery-cookie/jquery.cookie.js') }}" type="text/javascript"></script>
> -    <script type="text/javascript" src="/static/new/js_external/json2.js"></script>
>
> @BonfaceKilz  Json is now included in the js language so I don't think we need the library.

Ah yes! I see. That never occurred to me. Then in
this case this PR LGTM! I'll have this merged. I
don't think there's anything standing out. Have a
look at the other deletions PR; I left some
comments there.

-- 
Bonface M. K. D4F09EB110177E03C28E2FE1F5BBAE1E0392253F
Humble GNU Emacs User / Bearer of scheme-y parens
Curator: <https://upbookclub.com> / Twitter: @BonfaceKilz

**Review by BonfaceKilz — APPROVED** [review]

**Pull request merged**